content/en/docs/setup/upgrade/canary/index.md

```diff
@@ -94,29 +94,20 @@ To upgrade the namespace `test-ns`, remove the `istio-injection` label, and add
 $ kubectl label namespace test-ns istio-injection- istio.io/rev=canary
 {{< /text >}}
 
-After the namespace updates, you need to restart the pods to trigger re-injection. One way to do
-this is using:
+After the namespace updates, you need to restart the pods to trigger re-injection.
+One way to restart all pods in namespace `test-ns` is using:
 
 {{< text bash >}}
 $ kubectl rollout restart deployment -n test-ns
 {{< /text >}}
 
-When the pods are re-injected, they will be configured to point to the `istiod-canary` control plane. You can verify this by looking at the pod labels.
-
-For example, the following command will show all the pods using the `canary` revision:
-
-{{< text bash >}}
-$ kubectl get pods -n test-ns -l istio.io/rev=canary
-{{< /text >}}
-
-To verify that the new pods in the `test-ns` namespace are using the `istiod-canary` service corresponding to the `canary` revision, select one newly created pod and use the `pod_name` in the following command:
+When the pods are re-injected, they will be configured to point to the `istiod-canary` control plane. You can verify this by using `istioctl proxy-status`.
 
 {{< text bash >}}
-$ istioctl proxy-status | grep ${pod_name} | awk '{print $7}'
-istiod-canary-6956db645c-vwhsk
+$ istioctl proxy-status | grep "\.test-ns "
```

Comment:
It seems the output, istiod-canary-...,  was removed.

In the next paragraph we mention looking at the output and there is none shown in this document.

Reply:
Yes.
The output will show all pods in user's namespace, and is not determinate.
So i omit the output.